app/src/main/java/com/github/se/icebreakrr/ui/sections/Settings.kt

```diff
@@ -222,13 +228,17 @@ fun ToggleOptionBox(
               .testTag(label),
       colors =
           CardDefaults.cardColors(
-              containerColor = MaterialTheme.colorScheme.secondaryContainer,
-              contentColor = MaterialTheme.colorScheme.onPrimary),
-      elevation = CardDefaults.cardElevation(defaultElevation = CARD_ELEVATION)) {
+              containerColor = MaterialTheme.colorScheme.background,
+              contentColor = MaterialTheme.colorScheme.onSecondary)) {
         Row(
             modifier = Modifier.fillMaxWidth().padding(CARD_PADDING),
             verticalAlignment = Alignment.CenterVertically) {
-              Text(label)
+              Icon(
+                  imageVector = Icons.Filled.Place,
+                  contentDescription = "",
+                  modifier = Modifier.padding(end = 8.dp) // Add padding to the right of the icon
```

Comment:
Nitpick: magic number

Reply:
I hope it's the last one, sorry for the malfunction 🤖